src/components/symptoms/test.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import "./style.css";
+
+import Button from "@material-ui/core/Button";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormControl from "@material-ui/core/FormControl";
+
+class Symptoms extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: "Nej",
+      firstPage: true,
+      isFirstPageValid: false,
+      
+    };
+
+    this.onRadioChange = this.onRadioChange.bind(this);
+    this.handleFirstPageClick = this.handleFirstPageClick.bind(this);
+    this._handleClick = this._handleClick.bind(this);
+  }
+
+  onRadioChange = (e) => {
+
+    const { target } = e; 
+    const value = target.value;
+    
+    this.setState({
+      value,
+      isFirstPageValid: value === "Ja", 
+    });
+  };
+
+  handleFirstPageClick = () => {
+    this.setState((prevState) => ({ firstPage: !prevState.isFirstPageValid }));
+  };
+
+  _handleClick() {
+    const { firstPage } = this.state;
+    const { onClick } = this.props;
+
+    if (firstPage) this.handleFirstPageClick();
+    else if(this.state.value === 'Ja')
+    onClick();
```

Comment:
not wrong but I would shy away from using _else if_ if you're not gonna have an _else_ clause as well. If you wanna be specific about the condition just use another _if_ statement and if not you can use _else_

Reply:
Hi Filip, I tried but I couldn't solve this . Maybe you can help me with this. Thank you.